eb

```diff
@@ -127,7 +134,7 @@ export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
 NEW_ARGS=()
 for argument in "$@"; do
 	if [[ "$CURRENT_USER" == "ebuser" ]]; then
-		if [[ $argument =~ --inject-checksums* || $argument =~ --robot* || "$argument" == "-r" ]]; then
+		if [[ $argument =~ --inject-checksums* || $argument =~ --robot$ || "$argument" == "-r" ]]; then
```

Comment:
why should this only apply to `--robot` at EOL?

Reply:
It sends `--robot-path` (but without `--robot`) to the job script